norfair/metrics.py

```diff
@@ -10,21 +10,22 @@
 
 class InformationFile:
     def __init__(self, file_path):
+        self.path = file_path
         with open(file_path, "r") as myfile:
             self.file = myfile.read()
```

Comment:
`self.file` is only used here, so we can remove it from `self` and just name it `file`.